src/test/ts/commands/StatusTest.ts

```diff
@@ -17,9 +18,22 @@ const check = async (dir: string, expected: Object) => {
 };
 
 describe('Status', () => {
+  let verdaccio: cp.ChildProcess;
+  let address: string;
+
+  before(async () => {
+    const v = await TestUtils.startVerdaccio();
```

Comment:
This was needed because the status command does `npm dist-tag ls packageName` and newer versions of NPM won't work with the dummy address that was being used here. We needed a real NPM registry.